firmware/quadruna/BMS/src/app/states/app_chargeState.c

```diff
@@ -1,18 +1,46 @@
+#include "app_chargeState.h"
 #include "states/app_allStates.h"
 #include "io_airs.h"
 #include "io_charger.h"
 
 // 0.05C is standard for a boundary to consider full charge
-#define CURRENT_AT_MAX_CHARGE (0.05f * C_RATE_TO_AMPS)
+#define C_RATE_FOR_MAX_CHARGE (0.05f)
 #define MAX_CELL_VOLTAGE_THRESHOLD (4.15f)
+#define CURRENT_AT_MAX_CHARGE (C_RATE_FOR_MAX_CHARGE * C_RATE_TO_AMPS)
+
+static uint16_t sixteenByteEndianSwap(uint16_t can_signal)
```

Comment:
Technically it's two bytes, but maybe a function name more akin to the use case would be better eg:

```suggestion
static uint16_t canMsgEndianSwap(uint16_t can_signal)
```